AdyenDropIn/Views/ModalToolbar.swift

```diff
@@ -161,12 +161,9 @@ internal final class ModalToolbar: UIView, AnyNavigationBar {
             button = UIButton(type: UIButton.ButtonType.custom)
             button.setImage(image, for: .normal)
         default:
-            if #available(iOS 13.0, *) {
-                button = UIButton(type: UIButton.ButtonType.close)
-                button.widthAnchor.constraint(equalTo: button.heightAnchor).isActive = true
-            } else {
-                return legacy()
-            }
+            let legacyButton = legacy()
+            legacyButton.titleLabel?.font = UIFont(name: "SF Pro Text", size: 17.0)
```

Comment:
I am curious why we need this hardcoded font? 
Could it be `.systemFont(ofSize: 17)`
Could it be in configurable from `style`?

Reply:
This I have removed.